var/spack/repos/builtin/packages/hapcut2/package.py

```diff
@@ -13,15 +13,33 @@ class Hapcut2(MakefilePackage):
 
     homepage = "https://github.com/vibansal/HapCUT2"
     git      = "https://github.com/vibansal/HapCUT2.git"
+    maintainers = ['snehring']
 
+    version('v1.3.3', commit='c6481d5fd0618dc3e82b2eb8c2b4835d9a4f6da7')
```

Comment:
It's unfortunate that that commit does not provide any indication it is an actual release and the last actual release was `v1.3.1` (2021 Jul 16).  Why did you choose `v1.3.3`?

Any way to get the repository maintainers to make a release?

Reply:
Same commit as that release. I think they just mislabeled it.

Reply:
The more I think about it though maybe 1.3.1 is correct and the tags are just extraneous.